third_party/xla/xla/service/gpu/fusions/triton/dot_algorithms_test.cc

```diff
@@ -1146,6 +1146,9 @@ CHECK-NOT: mma.sync.aligned.{{.*}}.row.col.f32.tf32.tf32.f32
 }
 
 TEST_F(TritonAlgorithmTest, Algorithm_BF16_BF16_F32_X3) {
+  if (std::holds_alternative<se::RocmComputeCapability>(GpuComputeComp())) {
+    GTEST_SKIP() << "Triton currently disabled on ROCM.";
+  }
```

Comment:
when you skip one subtest, could you leave a comment to indicate the date like this? https://github.com/ROCm/tensorflow-upstream/pull/2704/commits/f79915189d42c069fbc993874587bae28579e339